src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -207,7 +207,7 @@ public function createTable(Table $table, array $columns = [], array $indexes =
                    ->setIdentity(true);
 
             array_unshift($columns, $column);
-            if (isset($options['primary_key'])) {
+            if (isset($options['primary_key']) && $options['id'] !== $options['primary_key'] && $options['id'] !== $options['primary_key'][0]) {
```

Comment:
$options['id'] !== $options['primary_key'][0]

Doesnt that mean you could have multiple values and it wouldn't find this?
Wouldnt it be cleaner to say

    if (isset($options['primary_key']) && $options['id'] !== (array)$options['primary_key']) {

Reply:
You are right on your comment, though the `(array)` cast was on the wrong side here. Fixed.